config/environments/test.rb

```diff
@@ -82,6 +82,14 @@
   # To control the debugger turing testing
   config.activate_debugger = false
 
+  # Enable stdout logger
+  config.logger = Logger.new($stdout)
+
+  # Set log level
```

Comment:
Would you mind including more informative comments.  One can clearly see that it is setting something called log_level without your comment... however that tells me nothing about what it does! :)

Reply:
@pellaea I'll try to track down why i added these - i've been trying to comment everything copiously but missed this one. They may have been generated by a graphql gem, on the other hand.  🤔